.travis.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
     # Gitea
     - stage: test
-      env: GITEA_VERSION=1.6
```

Comment:
Hmm. Do we continue to support 1.6? If yes, then add an entry for `latest` here, do not remove 1.6.

Reply:
Oh, I'd better comment this on #210 :(

Reply:
OK, I'll add it back in.